src/shims/unix/foreign_items.rs

```diff
@@ -60,6 +60,21 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 let result = this.open(args)?;
                 this.write_scalar(Scalar::from_i32(result), dest)?;
             }
+            "epoll_create1" => {
```

Comment:
FYI there are two syscalls, `epoll_create` and `epoll_create1` that are pretty much functionally equivalent and should share an implementation.

https://man7.org/linux/man-pages/man2/epoll_create1.2.html